### DISCUSSION TITLE
Accessibility improvements

### DISCUSSION THREAD

**Discussion by Olyno** [logto-io/logto#1581] [Ideas]
Hi 👋🏻 

Some parts of the admin panel are not fully accessible. I'm thinking in particular of the registration form which is not sent at "submit", but at the click of the button. I think this can be improved.

So far, I've only seen the problem with the form, and I'm not good enough at accessibility to spot the other problems. I was thinking of making this an issue, but a discussion is more appropriate. If new people find accessibility issues, feel free to report them in this thread!

Linked to issue: #1577

Here are some resources about accessibility:
 - https://web.dev/accessible/

**Comment by gao-sun** [comment]
@simeng-li will follow this

**Reply by fleuraly** [reply]
Hey @Olyno , appreciate your feedback! Just wanna clarify, are you referring to the manually click to submit the form thing? If so, it is in our fix list :)

**Comment by simeng-li** [comment]
Good call #1557 is addressed. We will continuously integrate those a11y features.

**Reply by Olyno** [reply]
Not related to the registration form, but the "Reminder" modal can't be submitted with "Enter"

![image](https://user-images.githubusercontent.com/25107942/179657301-a3a9a386-560c-4742-b996-41968bb0afea.png)
